Rakefile

```diff
@@ -4,3 +4,52 @@ require "rspec/core/rake_task"
 RSpec::Core::RakeTask.new(:spec)
 
 task :default => :spec
+
+namespace :openapi do
+  IBM_CLOUD_API_GEMS = %w[ibm_cloud_power ibm_cloud_vpc ibm_cloud_iam ibm_cloud_resource_controller]
```

Comment:
Recommend making this one-per-line and alphabetical (that way adding new ones makes the diff cleaner)

Reply:
Can this just be `IBM_CLOUD_API_GEMS = IBM_CLOUD_OPENAPI_JSON.keys`?